scripts/build.sh

```diff
@@ -1,18 +1,19 @@
-#! /usr/bin/bash
+#!/bin/sh
 
 set -e
 
 yarn bootstrap
 yarn clean:lib
-yarn update:version
 yarn gen:version
```

Comment:
This line must followed by line#7 since this generates a file that contains the current releasing version information, so that I would say the you could break this into 3 pieces of code.
`pre-build` `build` `post-build` which co-responds to:

1. `prepare` stage 
2. build` stage
3. `publish` stage.

in stage `prepare` we need to collect informations from the environment variables to generate correct metadata.
in stage `build` we do what it means to do, as well as  `publish`